app/models/person.rb

```diff
@@ -341,6 +347,10 @@ def self.find_or_fetch_by_identifier(diaspora_id)
     DiasporaFederation::Discovery::Discovery.new(diaspora_id).fetch_and_save
 
     by_account_identifier(diaspora_id)
+
```

Comment:
Layout/EmptyLinesAroundExceptionHandlingKeywords: Extra empty line detected before the `rescue`.